cdap-client/src/main/java/co/cask/cdap/client/config/ConnectionConfig.java

```diff
@@ -66,10 +67,11 @@ public ConnectionConfig(Id.Namespace namespace, String hostname, int port, boole
     this.hostname = hostname;
     this.port = port;
     this.sslEnabled = sslEnabled;
+    this.username = System.getProperty("user.name");
```

Comment:
The purpose of having a username in the CLI prompt was for when interacting with a secure CDAP cluster, so that the user knows who they've been authenticated as. This will be useful when we implement per-user permissions and probably also useful now, just so the user knows who they are. Why are we showing the "user.name" property instead?